tests/functional/DatadirTest.php

```diff
@@ -23,7 +23,8 @@ public function testRun(): void
                     [
                         'name' => 'queries',
                         'resource' => 'queries',
-                        'restrictionFilter' => '{"dateFrom":"-1 day","dateTo":"-1 day"}',
+                        'restrictionFilter' => '{"dateFrom":"'.getenv('SKLIK_DATE_FROM')
+                            .'","dateTo":"'.getenv('SKLIK_DATE_TO').'"}',
```

Comment:
to nechapu tuhle zmenu, to prace musi fungovat i s tom -1 day, ne?

Reply:
Právě že to na tom padalo. Je to testovací účet od Skliku co jsme dostali víceméně neoficiálně, tak do toho nemůžu moc lézt, ale řekl bych že tam neexistuje žádný account, který by měl včera nějaký rozjetý kampaně. Proto jsem tam dal ty pevný data u kterých vím, že tam nějaký jsou.

Reply:
aha, nj, tak asi jo :)